thorlcr/slave/slave.cpp

```diff
@@ -52,7 +52,6 @@ MODULE_INIT(INIT_PRIORITY_STANDARD)
 ProcessSlaveActivity::ProcessSlaveActivity(CGraphElementBase *container) : CSlaveActivity(container), threaded("ProcessSlaveActivity", this)
 {
     processed = 0;
```

Comment:
picky: I'd have moved this to also be initialized at declaration time now lastCycles is.